src/java/main/org/apache/zookeeper/Version.java

```diff
@@ -20,7 +20,7 @@
 
 public class Version implements org.apache.zookeeper.version.Info {
 
-    public static int getRevision() {
+    public static String getRevision() {
```

Comment:
Since this is not an exposed interface it is OK to change the signature. But, I'm curious to know that any other project or users depending on the revision number.